polkadot/runtime/rococo/src/lib.rs

```diff
@@ -234,6 +235,32 @@ impl PrivilegeCmp<OriginCaller> for OriginPrivilegeCmp {
 	}
 }
 
+/// Dynamic params that can be adjusted at runtime.
+#[dynamic_params(RuntimeParameters, pallet_parameters::Parameters::<Runtime>)]
+pub mod dynamic_params {
+	use super::*;
+
+	#[dynamic_pallet_params]
+	#[codec(index = 0)]
+	pub mod nis {
+		use super::*;
+
+		/// The target of active total issuance fot the non-interactive-staking pallet.
+		#[codec(index = 0)]
+		pub static Target: Perquintill = Perquintill::zero();
```

Comment:
how about adding one more parameter that will have some non-zero, not none initial value? and plus we can have one parameter under root origin and another under some custom origin like staking admin.

Reply:
Yea good idea - just for testing all cases. Will add some more.